indexes/private.js

```diff
@@ -20,15 +20,13 @@ module.exports = function (dir, keys) {
   const encryptedFile = path.join(indexesPath(dir), 'encrypted.index')
   const canDecryptFile = path.join(indexesPath(dir), 'canDecrypt.index')
 
-  function save(filename, seq, arr, cb) {
+  function save(filename, arr) {
     const buf = toBuffer(fic.compress(arr))
     const b = Buffer.alloc(4 + buf.length)
-    b.writeInt32LE(seq, 0)
```

Comment:
It seems we were writing an Obv in the buffer! that seems wrong, right? I made it obv.value instead. does this fix some weird bugs?

Reply:
Oh 🤦, I was changing that variable back and forth and in shared log PR and didn't get it properly fixed everywhere it seeems. Nice catch.